engine/src/main/java/org/terasology/rendering/dag/FinalPostProcessingNode.java

```diff
@@ -143,14 +145,14 @@ private void renderFinalMonoImage() {
     // TODO: have a flag to invert the eyes (Cross Eye 3D), as mentioned in
     // TODO: http://forum.terasology.org/threads/happy-coding.1018/#post-11264
     private void renderFinalStereoImage(RenderingStage renderingStage) {
-        if (postProcessor.isNotTakingScreenshot()) {
-            sceneFinal.bind();
-        } else {
-            ocUndistorted.bind();
-        }
-
         switch (renderingStage) {
             case LEFT_EYE:
+                if (postProcessor.isNotTakingScreenshot()) { // TODO: verify if this works
```

Comment:
I just tried now, unfortunately it doesn't work if the if-clause is moved inside the case:

<img width="1264" alt="screen shot 2016-07-04 at 11 32 16" src="https://cloud.githubusercontent.com/assets/136392/16555295/b2bdbbd2-41dc-11e6-820a-d65228a60f91.png">

Previously it was:
<img width="1264" alt="screen shot 2016-07-04 at 11 31 16" src="https://cloud.githubusercontent.com/assets/136392/16555321/cacc12b4-41dc-11e6-8062-a1ecff0076f6.png">


I think it is better to revert that change and leave it as is.